src/common/images/opensuse.svg

```diff
@@ -1 +1,15 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 189.05 87.61">
```

Comment:
Can you run the file through https://jakearchibald.github.io/svgomg/ to remove unnecessary metadata?